docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -352,6 +352,8 @@ a file called `aws-values.yaml` and write the values you've chosen above to it:
 
 <ScopedBlock scope={["oss", "cloud"]}>
 
+<Tabs>
+<TabItem label="cert-manager">
```

Comment:
The cert-manager/AWS Cert Manager nomenclature is extra confusing. Could we use alternative terms like "Using LetsEncrypt" and "Using ACM" instead?

Reply:
We already do the same thing in the section above so I was just trying to be consistent:
![image](https://user-images.githubusercontent.com/897821/217087614-d813a8de-1562-4937-b9d0-e490a8750d1a.png)

Overall I think `cert-manager` might be clearer as I wouldn't want to get people mixed up with Teleport's built-in ACME support? 😖

Reply:
I'm not an SME in this space, but I've dabbled, and IMHO "cert-manager" is its own unique term and easy enough to conceptually differentiate from anything else. And "AWS Certificate Manager" is clearer than an acronym like "ACM". IMHO, keep it as is.